### ISSUE TITLE
Repro gap on HIV 5-shot (OFA-joint-lr)

### ISSUE THREAD

**Issue by yiming421** [LechengKong/OneForAll#29]
Hi authors, thanks for your great work!

  I’m trying to reproduce the HIV 5-shot result in Table 17 (`OFA-joint-lr`, reported `63.58±1.81` AUC), but I consistently get ~0.50 AUC.

  ## Version
  - Repo: `main`
  - Commit: `9045ccd`

  ## Command used
  ```bash
  python run_cdm.py --override lr_all_config.yaml num_epochs 30 load_best false > lr_joint.out 2>&1

  ## Confirmed runtime config 

  - task_names=['arxiv_fs', 'FB15K237_fs', 'mol_fs']
  - num_epochs=30
  - load_best=False
  - d_multiple=[10,10,0.5]
  - d_min_ratio=[5,5,0.5]

  ## Observed result

  - test_chemhiv_fs_25/auc: 0.52

  ## Questions

  1. Is the above command/config the intended setup for reproducing Table 17 HIV 5-shot (OFA-joint-lr)?
  2. Are there any additional required settings (specific commit, seed protocol, preprocessing/cache reset, checkpoint initialization, etc.) not captured in README/configs?

  Happy to run any exact command you recommend and report back.
